script/generate-cask.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env brew ruby
+require 'cask/cask'
+directories = ["_data/cask", "api/cask", "cask"]
+FileUtils.rm_rf directories
+FileUtils.mkdir_p directories
+
+Cask::Cask.each do |c|
+  json_filename = "#{File.basename(c.sourcefile_path,'.rb')}.json"
+  IO.write("_data/cask/#{json_filename}", JSON.pretty_generate(c.to_h))
+  FileUtils.cp "_api_formula.json.in", "api/cask/#{json_filename}"
```

Comment:
Rename this file now maybe?

Reply:
Like create a new file `_api_cask.json.in`? The current file is a template that exists in the repo.

Reply:
Yes, either a new file or rename the other to just `_api.json.in` to reflect that it's used for formulae and casks.

Reply:
I'm not sure where the json is served - is the variable `formula_json` going to be accurately filled for casks? Or would we need to change it to `cask_json`?

Reply:
I suggest testing it locally and seeing what works. New files may be needed and when they are not it's worth renaming anything that references a formula to cask instead.

Reply:
I created a `_layouts/cask_json.json` [gist](https://gist.github.com/cmbernard333/3a3d445458bb15128dc86ffd2d6c4744) and updated it to pull the correct information in `_data/cask/<cask>` but it seems json elements in the analytics are not named 

```
    {
      "number": 9,
      "cask": "google-chrome",
      "count": "11,375",
      "percent": "1.76"
    },
```

So it will not be able to pull in the analytics information.

Reply:
@cmbernard333 That's done by a separate analytics job for homebrew-core: https://github.com/Homebrew/formulae.brew.sh/blob/master/Rakefile#L58-L66

It may make sense to just skip it for now.

Reply:
Ok. That should be it then. I need to create an associated PR for `homebrew-cask` to pull down and run `rake cask`.

Reply:
Here's the PR [#58112](https://github.com/Homebrew/homebrew-cask/pull/58112) for `hombrew-cask` but it can't be updated to use the new rake target until this PR is merged.